packages/mui-material/src/ToggleButton/ToggleButton.js

```diff
@@ -38,40 +38,48 @@ const ToggleButtonRoot = styled(ButtonBase, {
 })(({ theme, ownerState }) => {
   const selectedColor =
```

Comment:
You can create `selecteColorChannel` like @mnajdova suggested:
```js
// ToggleButton.js
import { colorChannel } from '@mui/system';

...
  let selectedColor = ownerState.color === 'standard'
      ? theme.palette.text.primary
      : theme.palette[ownerState.color].main;
  const selectedColorChannel = colorChannel(selectedColor);
  if (theme.vars) {
    selectedColor = ownerState.color === 'standard'
      ? theme.vars.palette.text.primary
      : theme.vars.palette[ownerState.color].main;
  }
```

Then replace the `alpha(...)` like this:
```js
backgroundColor: theme.vars ? `rgba(${selectedColor} / ${theme.palette.action.selectedOpacity}) : alpha(selectedColor, theme.palette.action.selectedOpacity),
```

Reply:
I have already updated the code as you and @mnajdova suggested.